site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1168,11 +1168,13 @@ public function getActiveUserIds(bool $instructor, bool $fullAccess, bool $limit
             $extra_join = ' INNER JOIN users ON courses_users.user_id = users.user_id ';
             $extra_where = ' OR users.user_access_level <= 2';
         }
+        $terms = ($term === '' ? 'courses.term' : "'$term'");
+        $courses = ($course === '' ? 'courses.course' : "'$course'");
         $result_rows = [];
         $this->submitty_db->query(
             "SELECT DISTINCT courses_users.user_id as user_id
                 FROM courses_users INNER JOIN courses
-                ON courses_users.term = courses.term AND courses_users.course = courses.course
+                ON courses_users.term = " . $terms . " AND courses_users.course = " . $courses . "
```

Comment:
This seems like a pretty clear opportunity for a SQL injection in the future.  It would be better to use a prepared statement for text which isn't static.